src/api/File.js

```diff
@@ -81,6 +85,51 @@ class File extends Item {
         this.successHandler(`${downloadBaseUrl}?${downloadUrlQuery}`);
     }
 
+    /**
+     * Polls a representation's infoUrl, attempting to generate a representation
+     *
+     * @param {FileRepresentation} representation - representation that should have it's info.url polled
+     * @return {Promise<FileRepresentation>} - representation updated with most current status
+     */
+    async generateRepresentation(representation: FileRepresentation): Promise<FileRepresentation> {
+        const infoUrl = getProp(representation, 'info.url');
+
+        const numOfTries = 4;
```

Comment:
Do these values need to be different than the default values of `retryNumOfTimes`? If so, do they need to be declared as separate variables? If so, should they be declared outside of the function as constants?

Reply:
Yup, these values don't need to be declared as separate variables. I'll move them into the function call.

There are a couple of discussions about the polling values above: https://github.com/box/box-ui-elements/pull/1486#discussion_r310803697 and https://github.com/box/box-ui-elements/pull/1486#discussion_r311165030. Given that some representations take longer to generate, I think it makes sense to set a value higher than the default `initialTimeout` specified in `retryNumberOfTimes()`.